package.json

```diff
@@ -139,6 +139,9 @@
       "global": {
         "lines": 40
       }
+    },
+    "moduleNameMapper": {
+      "^.+\\.(css|less)$": "<rootDir>/test/cssStub.ts"
```

Comment:
Jest doesn't know how to handle css files natively so this was needed because of these lines in `browser/register.ts`

```

import "./pages/error.css"
import "./pages/global.css"
import "./pages/login.css"
```